### PR TITLE
change warning check in RepeatReadTest

### PR DESCRIPTION
**Pull request by en-sc**

### CODE REVIEW

**Comment by en-sc** [comment]
Related to https://github.com/riscv/riscv-openocd/pull/871

File: debug/gdbserver.py [hunk 1]
```diff
@@ -912,10 +916,15 @@ def test(self):
         output = self.gdb.command(
             f"monitor riscv repeat_read {count} 0x{mtime_addr:x} 4")
         values = []
-        for line in output.splitlines():
-            # Ignore warnings
-            if line.startswith("Batch memory"):
-                continue
```
Comment (timsifive) [inline debug/gdbserver.py:918]: I still see an error at https://github.com/riscv/riscv-openocd/pull/871/files#diff-701864e2be3210275085c84b8f6fa7f9eb6171a2a18ca71638be42a155c08c63R3574 that would be caught by this. Are you sure it's correct to remove this warning?

Comment (en-sc) [reply debug/gdbserver.py:918]: It's a `LOG_TARGET_DEBUG`, so it's not reported to user (in a gdb session).

**Comment by en-sc** [comment]
Old warnings are also supported now, in accordance with [this comment](https://github.com/riscv/riscv-openocd/pull/871#issuecomment-1623962270) by @timsifive.

**Review by timsifive — APPROVED** [review]

**Pull request merged**